compiler/qsc_linter/src/tests.rs

```diff
@@ -0,0 +1,115 @@
+use crate::{linter::ast::run_ast_lints, Lint, LintLevel};
+use expect_test::{expect, Expect};
+use qsc_ast::{
+    assigner::Assigner,
+    ast::{self, NodeId},
+    mut_visit::MutVisitor,
+};
+use qsc_data_structures::language_features::LanguageFeatures;
+
+#[test]
+fn double_parens() {
+    check_ast(
+        "let x = ((1 + 2));",
+        &expect![[r#"
+            [
+                SrcLint {
+                    source: "((1 + 2))",
+                    message: "unnecesary double parentheses",
+                    level: Warning,
+                },
+            ]
+        "#]],
+    );
+}
+
+#[test]
+fn multiple_lints() {
```

Comment:
For completeness, could you add a test that _only_ test the division by zero lint?